frontend/src/app/main/ui/dashboard/sidebar.cljs

```diff
@@ -255,7 +255,7 @@
        [:p (tr "modals.leave-and-reassign.hint1" (:name team))]
 
        (if (empty? members)
-         [:p (tr "modals.leave-and-reassign.forbiden")]
```

Comment:
This change (and translation strings file also) should be reverted and treated separatelly. Because there are a lot of translations and we will need to propagate this change to our translations platform. I will take a note on my todo list to do it myself, but it should be removed from this MR.

In any case, ty for noting about this.